templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<p>Same comment here - can we move this sentence: "If you would like to test the features in Ark with an example dataset, run the cell below. It will download a dataset consisting of 11 FOVs with 22 channels. You may find more information about the example dataset in the <a href="https://github.com/angelolab/ark-analysis/raw/f2204a3a3358bd87a3c5b3a9be5f78480f714bcb/README.md#example-dataset" target="_blank">README</a>." to below the cell setting the <code>base_dir</code>  (so it's right before the explanation to overwrite_existing)? Can we also add a sentence saying "If you are using your own data, skip the cell below." to the end of that. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/856/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>